### PR TITLE
Now Menus react to buttons a bit nicer :3

### LINKED ISSUE #154: `Enter` behaves differently in mod's menus compared to OW's
We somehow need to discerne when `Enter` is used as `select` and when it's used as `save and exit`
Need to investigate how OWMenuInputModule works

### PR DESCRIPTION
**Pull request by TAImatem**
Partially fixes #154
Part 1, ModPopupInput (that is both FieldInput and ComboInput) is next
PS any other Input I should look at?

### CODE REVIEW

File: OWML.ModHelper.Input/ModCommandListener.cs [hunk 1]
```diff
@@ -75,7 +97,18 @@ private void Update()
                 {
                     OnTapped?.Invoke(command);
                 }
+                if (blockFlag)//damn you, Mobius Digital
+                {
+                    if (!(_wasPressed[command] || _isPressed[command]))
+                    {
+                        command.SetValue("_blockNextRelease", false);
```
Comment (amazingalek) [inline OWML.ModHelper.Input/ModCommandListener.cs:104]: var toBlock = ...;
command.SetValue(..., toBlock);

**Comment by amazingalek** [comment]
Looks great!

File: OWML.ModHelper.Input/ModCommandListener.cs (hunk 1 above)
Comment (amazingalek) [reply OWML.ModHelper.Input/ModCommandListener.cs:104]: (this removes the if)

**Comment by TAImatem** [comment]
weird that it didn't get merged automatically...

**Comment by amazingalek** [comment]
Please check for unused usings :-)

File: OWML.ModHelper.Input/ModCommandListener.cs (hunk 1 above)
Comment (TAImatem) [reply OWML.ModHelper.Input/ModCommandListener.cs:104]: done

**Comment by TAImatem** [comment]
pretty sure found them all, @amazingalek

**Pull request merged**